levels/base_level.tscn

```diff
@@ -49,6 +45,15 @@ func difficulty_up():
 	wait_time *= SPAWN_RATE_INCREASE
 "
 
+[sub_resource type="GDScript" id="GDScript_3xnds"]
+script/source = "extends Node2D
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
```

Comment:
Test comment: "_" on "_ready()" not needed

Reply:
Test